Makefile

```diff
@@ -29,6 +29,9 @@ postman:
 	# For example: make postman env=test token=<MY_TOKEN>
 	docker-compose -f docker-compose.test.yml run --rm postman_test test/postman_test/SSAS.postman_collection.json -e test/postman_test/ssas-local.postman_environment.json --global-var adminClientId=$(SSAS_ADMIN_CLIENT_ID) --global-var adminClientSecret=$(SSAS_ADMIN_CLIENT_SECRET)
 
+migrations-test:
+	docker-compose -f docker-compose.test.yml run --rm tests bash ops/migrations_test.sh
```

Comment:
Could this just be run via CLI from the SSAS container?  Is it pointed at the same DB in both cases?  If so, I would argue that `golang-migrate` can be removed from the tests image.

Reply:
No, the SSAS container points at the non-test DB.